### DISCUSSION TITLE
Install command

### DISCUSSION THREAD

**Discussion by Timothee-Cardoso** [scmn-dev/secman#9] [Ideas]
# Hi @abdfnx 

I've an idea, the idea is `install` command
like if I want to install package

1. clone the package
2. rm old secman
3. add the new secman with the new package

I hope to add this to **secman**

**Comment by abdfnx** [comment]
# weclome @Timothee-Cardoso 

wow, I'well try to add it to **secman**
thank you for this idea

**Comment by abdfnx** [comment]
Sorry Bro, but `install` command have many errors, maybe in the future it's well restore

Thanks you @Timothee-Cardoso for this great idea
